ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -700,10 +715,12 @@ const mstp = (state, {params: {dashboardID}}) => {
   const selectedCell = cell
 
   return {
+    script,
     sources,
     services,
     meRole,
     dashboard,
+    links: links.flux,
```

Comment:
since these links are specific to flux, how do you feel about naming it something more specific like `fluxLinks`?

Reply:
will do!